app/src/main/java/app/revanced/integrations/patches/LithoThemePatch.java

```diff
@@ -20,12 +18,17 @@ public class LithoThemePatch {
         -98492127 // video chapters list background
     };
 
+    // boolean used to check if shorts comment box is currently visible
+    public static boolean shortsCommentBox = false;
+
     // Used by app.revanced.patches.youtube.layout.theme.patch.LithoThemePatch
     public static int applyLithoTheme(int originalValue) {
         var isDarkTheme = ThemeHelper.isDarkTheme();
         
-        if ((isDarkTheme && anyEquals(originalValue, DARKCONSTANTS)) || (!isDarkTheme  && anyEquals(originalValue, WHITECONSTANTS)))
-                return 0;
+        if ((isDarkTheme && anyEquals(originalValue, DARKCONSTANTS)) ||
+            (!isDarkTheme && anyEquals(originalValue, WHITECONSTANTS)))
+                return shortsCommentBox ? -16777215 : 0;
```

Comment:
Please create a final field for this magic constant the same way it is being done with the other constants.